scripts/charge-custom-fields-upload/main.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import requests
+import concurrent.futures
+
+API_URL = "https://api.metronome.com"
+TOKEN = ""
```

Comment:
We should pass this as an env var, similar to this pattern [here](
https://github.com/Metronome-Industries/metronome-recipes/blob/main/scripts/add-plan-to-customer-with-price-adjustments/global-discount.py#L11-L13)